x/application/types/params.go

```diff
@@ -6,6 +6,8 @@ import (
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 )
 
+const NameMaxDelegatedGateways = "max_delegated_gateways"
```

Comment:
Some of the other params have a default beside the name. Why don't we have one here?

Reply:
We do, it's just declared in the `var` group below. I think that's the result of an ignite scaffolding template. When I added `num_blocks_per_session` manually, I grouped constants together.